src/gpg/eopen.py

```diff
@@ -0,0 +1,72 @@
+import gpg
+import os
+import sys
+import time
+import textwrap
+
+DEBUG= True if (os.environ['DEBUG']=='yes') else False
```

Comment:
Maybe it can be simplified like this:
`DEBUG = (os.environ['DEBUG'] == 'yes')`
Or maybe you don't need an extra variable since you can use the test condition above.